chevah/geck
# [#73] Add documenation for the documentation styleguide.

Scope
=====

Fixes #73 

Changes
=======

Add an initial page.


As a drive-by the theme was updated to Furo so that we have dart and light version.

Also, the project was updated to Python 3 and latest sphinx version.


How to try and test the changes
===============================

reviewers: @dumol @MohamedWasimMohamedAkram 

check that changes make sense

Documentation preview here https://chevah-geck--89.org.readthedocs.build/styleguide/documentation.html
